cmd/kind/create/cluster/createcluster.go

```diff
@@ -50,15 +52,25 @@ func NewCommand() *cobra.Command {
 	}
 	cmd.Flags().StringVar(&flags.Name, "name", "1", "cluster context name")
 	cmd.Flags().StringVar(&flags.Config, "config", "", "path to a kind config file")
+	cmd.Flags().Int32Var(&flags.ControlPlanes, "control-planes", 1, "number of control-plane nodes in the cluster")
+	cmd.Flags().Int32Var(&flags.Workers, "workers", 0, "number of worker nodes in the cluster")
```

Comment:
Maybe somewhere in the usage we can mention the conflict between these flags and config?